modules/reference/pages/connector-helm-spec.adoc

```diff
@@ -0,0 +1,633 @@
+= Redpanda Connectors Helm Chart Specification
+:description: Find the default values and descriptions of settings in the Redpanda Connectors Helm chart.
+
+image:https://img.shields.io/badge/Version-0.1.6-informational?style=flat-square[Version:
+0.1.6]
+image:https://img.shields.io/badge/Type-application-informational?style=flat-square[Type:
+application]
+image:https://img.shields.io/badge/AppVersion-v1.0.6-informational?style=flat-square[AppVersion:
+v1.0.6]
+
+This page describes the official Redpanda Connectors Helm Chart. In
+particular, this page describes the contents of the chart's
+https://github.com/redpanda-data/helm-charts/blob/main/charts/connectors/values.yaml[`values.yaml`
+file]. Each of the settings is listed and described on this page, along
+with any default values.
+
+For instructions on how to install and use the chart, including how to
+override and customize the chart's values, refer to the
+link:https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/kubernetes/k-deploy-connectors/[deployment documentation].
+
+'''''
+
+Autogenerated from chart metadata using
+https://github.com/norwoodj/helm-docs/releases/v1.11.0[helm-docs
+v1.11.0]
+
+== Source Code
+
+* https://github.com/redpanda-data/helm-charts
+
+== Requirements
+
+Kubernetes: `^1.21.0-0`
+
+== Settings
+
+=== https://artifacthub.io/packages/helm/redpanda-data/connectors?modal=values&path=auth[auth]
+
+Authentication settings. For details, see the
+https://docs.redpanda.com/docs/manage/kubernetes/security/sasl-kubernetes/[SASL
+documentation].
+
+*Default:*
+
+....
+{"sasl":{"enabled":false,"mechanism":"scram-sha-512","secretRef":"","userName":""}}
+....
+
+=== https://artifacthub.io/packages/helm/redpanda-data/connectors?modal=values&path=auth.sasl.mechanism[auth.sasl.mechanism]
+
+The authentication mechanism to use for the superuser. Options are
+`scram-sha-256` and `scram-sha-512`.
+
+*Default:* `"scram-sha-512"`
+
+=== https://artifacthub.io/packages/helm/redpanda-data/connectors?modal=values&path=auth.sasl.secretRef[auth.sasl.secretRef]
+
+A Secret that contains your SASL user password.
```

Comment:
Is the initcap here specific to k8s/Helm chart? Otherwise, I see no reason to capitalize.